docs/en/reference/events.rst

```diff
@@ -121,6 +121,40 @@ Now you can test the ``$eventSubscriber`` instance to see if the
         echo 'pre foo invoked!';
     }
 
+Events Overview
+---------------
+
++-----------------------------+-----------------------+
+| Event                       | Dispatched by         |
++=============================+=======================+
+| ``preRemove``               | ``$em->remove()``     |
++-----------------------------+-----------------------+
+| ``postRemove``              | ``$em->flush()``      |
++-----------------------------+-----------------------+
+| ``prePersist``              | ``$em->persist()``    |
+|                             | on *initial* persist  |
++-----------------------------+-----------------------+
+| ``postPersist``             | ``$em->flush()``      |
++-----------------------------+-----------------------+
+| ``preUpdate``               | ``$em->flush()``      |
++-----------------------------+-----------------------+
+| ``postUpdate``              | ``$em->flush()``      |
++-----------------------------+-----------------------+
+| ``postLoad``                | Loading from database |
++-----------------------------+-----------------------+
+| ``loadClassMetadata``       | ?                     |
```

Comment:
This is hard to point out. It is triggered when metadata is loaded, which happens in reading methods of the entity manager like `find()`, `getReference()` and `getPartialReference()` when it is loaded for the first time. Of course it is also triggered when `$metadataFactory->getMetadataFor()` is called directly. 

If this table is focused on the entity manager, maybe those three $em methods can be added with the hint of an initial load.

It is also triggered on `$em->clear('Foo')`, but the argument for `clear()` is deprecated and will be removed.

Reply:
> maybe those three $em methods can be added with the hint of an initial load.

Does "initial" mean (a) for the first time in the current request, or (b) for the first time *ever* (i.e. until the database structure is changed)? If (a), then there's almost no (?) difference to what I wrote for `postLoad` (="Loading from database"), since you probably don't load the same entity 20 times in the same request...

Reply:
@SenseException